lib/scsi_defs.h

```diff
@@ -80,5 +88,60 @@ namespace scsi_defs {
     uint8_t vendorSpecific : 2;
   } ControlByte;
 
+  // SCSI Reference Manual Table 148 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  enum class SelectReport : uint8_t {
+    kRestrictedMethods = 0x0,
+    kWellKnown = 0x1,
+    kAllLogical = 0x2
+  };
+
+  // SCSI Reference Manual Table 147 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  struct ReportLunsCommand {
+    OptCode optCode = OptCode::kReportLuns;
+    uint8_t reserved1[1];
+    SelectReport selectReport;
+    uint8_t reserved2[3];
+    uint32_t allocLength;
+    uint8_t reserved3[1];
+    ControlByte controlByte;
```

Comment:
So far we've been explicitly declaring the number of bits each field is using. You can refer to #16